test/module-path-tests/src/main/java/module-info.java

```diff
@@ -30,5 +30,4 @@
 
     requires org.slf4j;
     requires slf4j.simple;
-    requires com.fasterxml.jackson.core;
```

Comment:
Hmm, does it mean this change would break customers who use the java SDK via module path and do not use Jackson for anything else?

Reply:
If they're defining "requires com.fasterxml.jackson.core" (or literally making any other in-code reference to Jackson) without including it themselves (or another dependency that does so), and we remove Jackson, then yes they will break. 

In general people should not rely on transitive dependencies without directly depending on it themselves. We have the maven dependency plugin to make sure we don't do that. I'm sure some people are, though, but there's nothing we can do about that. "We'll only break people that are already broken" is the name of the game here, I think.

Reply:
I think currently they do have to add `requires com.fasterxml.jackson.core;` otherwise they'll get `java.lang.NoClassDefFoundError: com/fasterxml/jackson/core/JsonFactory`; That's why we have it here in the first place. (If we had module-info in the SDK specifying `requires transitive com.fasterxml.jackson.core;`, then customers wouldn't need to.)

I don't think this is a blocker, but we should probably make it clear in the changelog entry.